site/src/docs/components/cookie-consent/api.mdx

```diff
@@ -0,0 +1,135 @@
+---
+slug: '/components/cookie-consent/api'
+title: 'CookieConsent - API'
+---
+
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## API
+
+### Component properties
+
+Common texts, groups and cookie data can be found in <Link href="">HDS-provided content file</Link>. The file contains all user interface strings, language options, descriptions for common cookie groups, and details about common Helsinki cookies. By default, this content file is used. You can also override parts of it and add your project-specific groups and cookies. 
```

Comment:
Missing Link href. Do we wish that user could download the HDS provided content file?

Reply:
Fixed